.gitignore

```diff
@@ -7,3 +7,4 @@ build
 
 # Ignore lock files in examples for now
 examples/**/yarn.lock
+.docusaurus
```

Comment:
We should be able to remove that entry, since it's present in `website/.gitignore`
```suggestion
```